back/engines/commercial/multi_tenancy/lib/tasks/core/add_last_survey_pages.rake

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
```

Comment:
You don't have to review the FE files- that stuff is just fixes for feedback from Rob that I accidentally pushed to the wrong branch